examples/code/audio_player/audio_player.cpp

```diff
@@ -88,7 +88,8 @@ namespace
 
         pen::jobs_create_job(put::audio_thread_function, 1024 * 10, nullptr, pen::e_thread_start_flags::detached);
         renderer_state_init();
-        put::dev_ui::init();
+        ImGuiStyle &style = ImGui::GetStyle();
```

Comment:
if the default arg to `dev_ui::init` is initialised to nullptr then this code doesn't need to change

Reply:
I was thinking that.

Reply:
I think we can revert these changes, since the init will now use the default. No need to merge un-necessary code

Reply:
to be clear I expect to see no changes whatsoever to this file in this pull request. You do not need to pass `style` into `put::dev_ui::init` because the default arg should handle it.